tox.ini

```diff
@@ -1,10 +1,49 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python-without-pypy
 [tox]
-envlist =
-    py27,py33,py34
+envlist = lint,
+          py27,
+          py35,
+          py36,
+          py37,
+          py38,
+          coverage
 
 [testenv]
+usedevelop = true
 deps =
-    zope.testing
     zope.testrunner
 commands =
-    zope-testrunner --test-path=src --auto-color --auto-progress
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+commands =
+    flake8 src setup.py
+    check-manifest
+
+[testenv:coverage]
+basepython = python3
+deps =
+    coverage
+    zope.testrunner
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report
```

Comment:
I would prefer `coverage report -m`, but I guess I should address these comments to the zopefoundation/meta repository that has the template?

Reply:
Fixed in zopefoundation/meta.

We are currently working on the template trying to migrate some packages to gain experience and to find the best and agreed practices.